src/Network/Http/Response.php

```diff
@@ -388,7 +388,7 @@ protected function _getJson()
             return $this->_json;
         }
         $data = json_decode($this->_body, true);
-        if ($data) {
+        if (is_array($data)) {
```

Comment:
Maybe everything except null?

    NULL is returned if the json cannot be decoded or if the encoded data is deeper than the recursion limit.